snapshot-labs/sx-monorepo
# feat: add members space page

### Summary

Add a new space page and item in the space menu called "Members". 

Closes: #960

### How to test

1. Open a space with members
2. Verify `Members` navigation item is visible
3. Open `Members`
4. If there are no members (admin, moderator, author) the navigation item is not rendered
